### DISCUSSION TITLE
kanister-kubectl docker image maintenance

### DISCUSSION THREAD

**Discussion by PrasadG193** [kanisterio/kanister#1362] [General]
As of now, we can see multiple `kanister-kubectl` images in the ghcr registry
- [ghcr.io/kanisterio/kanister-kubectl](https://github.com/orgs/kanisterio/packages/container/package/kanister-kubectl) - we don't build and publish this as a part of the release pipeline
- [ghcr.io/kanisterio/kanister-kubectl-1.18](https://github.com/orgs/kanisterio/packages/container/package/kanister-kubectl-1.18)

Going forward, in order to support recent versions of kubectl, we need to decide on the consistent image format which would be easier to maintain and extensible.

**Comment by pavannd1** [comment]
I think we should use [ghcr.io/kanisterio/kanister-kubectl](https://github.com/orgs/kanisterio/packages/container/package/kanister-kubectl) and keep it updated to the latest version of kubectl.
For special requirements, we can build separate images like [ghcr.io/kanisterio/kanister-kubectl-1.18](https://github.com/orgs/kanisterio/packages/container/package/kanister-kubectl-1.18)
